scripts/validate.js

```diff
@@ -6,51 +6,71 @@ const { join } = require('path');
 
 const { listDir } = require('hexo-fs');
 const { log } = hexo;
-const sizeOf = require('image-size');
+const sharp = require('sharp');
```

Comment:
Sharp requires node-gyp to build. Better enable node_modules cache in the CI

Reply:
Caching `node_modules` may [cause the test to fail](https://github.com/hexojs/hexo/pull/4630#issuecomment-782555535) and [is not recommended](https://github.com/hexojs/hexo/pull/4630#issuecomment-782555535)

`sharp` is also used in `scripts/responsive.js`, I believe this modification will not worsen the running time of the installation script